main/templates/footer.html

```diff
@@ -19,11 +19,13 @@ <h2><a href="https://mitbootcamps.zendesk.com/hc/en-us/requests/new">Contact Us<
         <![endif]-->
         <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
         <script>
-          hbspt.forms.create({
-            portalId: '{{ hubspot_portal_id }}',
-            formId: '{{ hubspot_footer_form_guid }}',
-            target: '#hs_footer_form'
-          });
+          if (typeof hbspt !== 'undefined') {
```

Comment:
Same as above
```suggestion
          if (!!hbspt) {
```

Reply:
I get `Uncaught ReferenceError: hbspt is not defined` again if I change the code to `!!hbspt` - probably because the variable is not declared if the external JS is blocked.